extension/js/common/browser/browser.ts

```diff
@@ -53,21 +57,28 @@ export class Browser {
     return array;
   };
 
-  public static openSettingsPage = async (path = 'index.htm', acctEmail?: string, page = '', rawPageUrlParams?: Dict<UrlParam>, addNewAcct = false) => {
+  public static openSettingsPage = async (
+    path = 'index.htm',
+    acctEmail?: string,
+    page = '',
+    rawPageUrlParams?: Dict<UrlParam>,
+    addNewAcct = false,
+    currentPageParams?: Dict<UrlParam>
```

Comment:
maybe we can use `rawPageUrlParams` for `useFullScreenCompose` parameter?

Reply:
Thank you - I have re-used the `rawPageUrlParams` parameter. I initially though it was only for iframe parameters but it was all clear to me know.